keyboards/cradio/info.json

```
@@ -8,8 +8,7 @@
       "pid": "0x3988",
       "device_version": "0.0.1"
     },
-    "processor": "atmega32u4",
-    "protocol": "atmel-dfu",
+    "development_board": "elite_c",
```

```suggestion
    "processor": "atmega32u4",
    "bootloader": "atmel-dfu",
    "pin_compatible": "promicro",
```
The cradio firmware only requires the promicro pins and the pcbs only use promicro footprints right? My concern is the development_board presets for elite-c could be changed in the future rendering this board incompatible with promicro converters.

---

Ah that makes sense — didn't know `pin_compatible` existed. Thanks for pointer.